resolver/request.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import requests
+from validator_collection import checkers
+
+from sceptre.resolvers import Resolver
+
+
+class Request(Resolver):
+    """
+    Resolve data from a REST API endpoint.
+    """
+
+    def _make_request(self, url):
+        """
+        Make a request to a REST API endpoint
+        :param url: The url endpoint reference
+        """
+        content = None
+        try:
+            response = requests.get(url)
+            content = response.text
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as err:
+            raise err
+        except requests.exceptions.RequestException as e:
+            raise e
```

Comment:
Why are you catching errors for no reason? Why not just remove these? The only reason to catch them is to do something with them, such as rewrapping them in a more user-friendly way.

Reply:
good point, removed catches.